app/uk/gov/hmrc/individualsmatchingapi/controllers/CitizenMatchingController.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsmatchingapi.controllers
```

Comment:
can the controller be uk.gov.hmrc.individualsmatchingapi.controllers.vp1 to distinguish with RAS one?

Reply:
or maybe rename the controller classes with P1 at the end

Reply:
@tom59 Unless you strongly feel about changing the package structure, I would rather rename the controller to CitizenMatchingControllerP1